test/probes/weakCrypto.spec.js

```diff
@@ -49,8 +49,11 @@ test("it should report a warning in case of `[expression]createHash(<weak-algo>)
 });
 
 test("it should NOT report a warning in case of `[expression]createHash('sha256')` usage", (tape) => {
```

Comment:
I just like to have a test like:

```js
const md5Usage = readFileSync(join(FIXTURE_PATH, "strongAlgorithms", "sha256.js"), "utf-8");
const { warnings: outputWarnings } = runASTAnalysis(md5Usage);
const code = `
    import c from 'crypto';
    const crypto = {
      createHash() {}
    }
    crypto.createHash('sha1');
`;
const { warnings: outputWarnings } = runASTAnalysis(code);

tape.strictEqual(outputWarnings.length, 0);
tape.end();
```

It checks that even if I have `crypto` imported in the scope imported and I use a `createHash` from another object, it won't raise an error. For me, it's the case I was not capable to catch before the tracer.